.yarnrc.yml

```diff
@@ -1,5 +1,11 @@
+enableGlobalCache: true
+
+httpTimeout: 300000
```

Comment:
What's the benefit of increasing the timeout from the default (60000) to 300000?

Reply:
not sure if it's needed, but saw that used in a couple of repos using yarn so might be some kind of best practice. fine to remove also.